### DISCUSSION TITLE
Real tabs

### DISCUSSION THREAD

**Discussion by PierreRust** [zim-desktop-wiki/zim-desktop-wiki#2054] [Ideas]
In the current interface, when using the "path bar" plugin,  a list of pages is displayed on top of the page content it-self. Depending on the mode used , this list can display recently changed pages, history or hierarchy.
This list allows jumping quickly from one page to another, in which it quite looks the 'tab' function found in many software like editors or browsers. 

How ever, it does not allow the user to really decide which tab should be displayed (no close button on tabs, no open in a tab on links, etc). Besides, it does not really look like tabs visually, which makes it difficult imho, to understand its function.

Is there some interest for a 'real-tab' function ? What would be the best approach for developing it, through a plugin  ?

**Comment by Qwertie-** [comment]
I quite like the tabs how they are. They serve as a useful way to quickly switch between the pages you visited most recently which 99% of the time is what you want to do.

**Comment by Nick222** [comment]
Is not it https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/974 ?

**Comment by camilajenny** [comment]
I like the current behavior, but ability to pin a tab would be an improvement.

**Reply by thomaswacker** [reply]
Yes that would be very good for me also.

**Comment by sojusnik** [comment]
@banan314 you kind of have it already with the Bookmarks Bar Plugin, see Zim's help.

**Reply by camilajenny** [reply]
it's the same in Joplin, but would be nice to have a possibility to move ctrl+tab like in a browser, otherwise usability is limited

**Reply by sojusnik** [reply]
You can go to the previous note by `alt+left` or even use the back/forward buttons on the mouse, when [this](https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/610) bug will be finally solved.
